src/Integration.Vsix.UnitTests/ErrorList/IssuesSnapshotTests _GetValue.cs

```diff
@@ -251,6 +252,15 @@ public void GetValue_Issue()
             GetValue(SonarLintTableControlConstants.IssueVizColumnName).Should().BeSameAs(issueViz);
         }
 
+        [TestMethod]
+        public void GetValue_SuppressionState_Is_SuppressionState()
+        {
+            issueViz.IsSuppressed = true;
+            GetValue(SuppressionsColumnHelper.SuppressionStateColumnName).Should().BeSameAs(SuppressionsColumnHelper.SuppressionState_Suppressed);
```

Comment:
@bigfluffycookie the first commit is your original one. The only changes I have made are to add the `SuppressionColumnHelper` and change your version to use those constants/properties instead of the VS version

e.g. here, using `SuppressionsColumnHelper.SuppresionStateColumnName` instead of `StandardTableKeyNames.SuppressionState`.